### ISSUE TITLE
Have access to adequate infrastructure

### ISSUE THREAD

**Issue by rasbt** [Benjamin-Lee/deep-rules#82]
**Have you checked the [list of proposed rules](https://github.com/Benjamin-Lee/deep-rules/issues?q=is%3Aissue+is%3Aopen+label%3Arule) to see if the rule has already been proposed?**

- [x] Yes

**Did you add yourself as a [contributor](https://github.com/Benjamin-Lee/deep-rules/blob/master/contributors.md) by making a pull request if this is your first contribution?**

- [x] Yes, I added myself or am already a contributor

**Feel free to elaborate, rant, and/or ramble.**

As far as I know, we don't have anything about infrastructure demands, yet. I.e., using DL in ones research will likely require access to multiple GPUs etc and is usually of higher computational demand compared to traditional models. So, researchers who relied on laptops and off-the-shelf desktop workstations for computational analyses should made aware that DL usually requires a bit more :).

This could potentially be mentioned in several places, e.g., 

2. Use traditional methods to establish performance baselines ([#41](https://github.com/Benjamin-Lee/deep-rules/issues/41), [#11](https://github.com/Benjamin-Lee/deep-rules/issues/11), [#10](https://github.com/Benjamin-Lee/deep-rules/issues/10))
- 3. Understand the complexities of training deep neural networks
- 6. Tune your hyperparameters extensively and systematically ([#42](https://github.com/Benjamin-Lee/deep-rules/issues/42), [#11](https://github.com/Benjamin-Lee/deep-rules/issues/11))

**Any citations for the rule?** (peer-reviewed literature preferred but not required)
- [DOI](doi.org/DOI_goes_here)

**Comment by evancofer** [comment]
It may be worth pointing out programs like [NVIDIA's GPU grant program](https://developer.nvidia.com/academic_gpu_seeding). These can be really useful for a small project or for new labs interested in DL. I believe some of the academic cloud credit programs can also be used for GPU or TPU compute time.

**Comment by rasbt** [comment]
Good point.  NVIDIA's GPU grant program is definitely nice and worth mentioning for smaller projects. Regarding the cloud credit, that's another thing to consider. I think AWS is mainly for teaching, but I am not sure. There are probably dozens of alternative programs that exist. We may want to mention this in general, because it could be immensely useful to researchers who think about employing DL but don't want to make a huge investment where it is unsure if DL really helps in their projects.

**Comment by fmaguire** [comment]
Seems best fit for the training complexities

**Comment by Benjamin-Lee** [comment]
I think that we should mention that free services exist but avoid going into details to make sure that the paper stays relevant for a longer time since these offerings have a way of disappearing.

Looking at you, [Travis/Idera](https://news.ycombinator.com/item?id=19218036).
